### DISCUSSION TITLE
$blog_id should not be initialized to true, isn't it?

### DISCUSSION THREAD

**Discussion by franck-paul** [JcDenis/dotclear#5] [General]
https://github.com/JcDenis/dotclear/blob/a20624e94d3106dcdb205c388728d7ff09ea23b4/src/Process/Admin/Handler/BlogPref.php#L35

Proposed:

```
    private $blog_id;
```

**Comment by JcDenis** [comment]
I think you're right. Except it should have a default empty value.
I don't fix type hint on all pages handlers yet, so several properties may have incorrect type.
